servo/connectors/kubernetes.py

```diff
@@ -766,6 +765,76 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return self.__str__()
 
+class HPA(KubernetesModel):
+
+    obj: kubernetes_asyncio.client.V1HorizontalPodAutoscaler
+
+    api_clients: ClassVar[Dict[str, Type]] = {
+        "preferred":kubernetes_asyncio.client.AutoscalingV1Api,
+        "v1":kubernetes_asyncio.client.AutoscalingV1Api,
+        "AutoscalingV2beta1Api":kubernetes_asyncio.client.AutoscalingV2beta1Api,
+        "AutoscalingV2beta2Api":kubernetes_asyncio.client.AutoscalingV2beta2Api,
```

Comment:
* Please include the full apiVersion in this mapping (eg. "autoscaling/v1")
* Please use the kubernetes apiVersion for this mapping, not the client's python type name
https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields

Reply:
Keys need to match the API version as would be specified in a `kubectl` command. Using the python kubernetes object names in _not_ correct.